test/katex-spec.js

```diff
@@ -2849,68 +2850,68 @@ describe("Unicode accents", function() {
             "\\~n" +
             "\\`o\\'o\\^o\\~o\\\"o" +
             "\\`u\\'u\\^u\\\"u" +
-            "\\'y\\\"y}");
+            "\\'y\\\"y}", strictSettings);
     });
 
     it("should support \\aa in text mode", function() {
-        expect("\\text{\\aa\\AA}").toParseLike("\\text{\\r a\\r A}");
-        expect("\\aa").toNotParse(new Settings({strict: true}));
-        expect("\\Aa").toNotParse(new Settings({strict: true}));
+        expect("\\text{\\aa\\AA}").toParseLike("\\text{\\r a\\r A}", strictSettings);
+        expect("\\aa").toNotParse(strictSettings);
+        expect("\\Aa").toNotParse(strictSettings);
     });
 
     it("should parse combining characters", function() {
-        expect("A\u0301C\u0301").toParseLike("Á\\acute C");
-        expect("\\text{A\u0301C\u0301}").toParseLike("\\text{Á\\'C}");
+        expect("A\u0301C\u0301").toParseLike("Á\\acute C", strictSettings);
+        expect("\\text{A\u0301C\u0301}").toParseLike("\\text{Á\\'C}", strictSettings);
     });
 
     it("should parse multi-accented characters", function() {
-        expect("ấā́ắ\\text{ấā́ắ}").toParse();
+        expect("ấā́ắ\\text{ấā́ắ}").toParse(strictSettings);
         // Doesn't parse quite the same as
         // "\\text{\\'{\\^a}\\'{\\=a}\\'{\\u a}}" because of the ordgroups.
     });
 
     it("should parse accented i's and j's", function() {
-        expect("íȷ́").toParseLike("\\acute ı\\acute ȷ");
-        expect("ấā́ắ\\text{ấā́ắ}").toParse();
+        expect("íȷ́").toParseLike("\\acute ı\\acute ȷ", strictSettings);
+        expect("ấā́ắ\\text{ấā́ắ}").toParse(strictSettings);
     });
 });
 
 describe("Unicode", function() {
     it("should parse negated relations", function() {
-        expect("∉∤∦≁≆≠≨≩≮≯≰≱⊀⊁⊈⊉⊊⊋⊬⊭⊮⊯⋠⋡⋦⋧⋨⋩⋬⋭⪇⪈⪉⪊⪵⪶⪹⪺⫋⫌").toParse();
+        expect("∉∤∦≁≆≠≨≩≮≯≰≱⊀⊁⊈⊉⊊⊋⊬⊭⊮⊯⋠⋡⋦⋧⋨⋩⋬⋭⪇⪈⪉⪊⪵⪶⪹⪺⫋⫌").toParse(strictSettings);
```

Comment:
Nice to see `toBuild` in the examples below!  Should this one also be `toBuild`?

Reply:
The `∉` entry is done via macro, not from `symbols.js`. It renders just fine. But KaTeX sends a console message that says, "No character metrics for '∉' in style 'Main-Regular'", which jest flags as an error in the build.

Is that a bug? Should it render without the console error?

Reply:
`\notin` produces the same error.  So that's a bug, though not necessarily for this PR.

Reply:
I was mistaken earlier when I said it was a macro. It is in fact in `symbols.js`.  Here's the line:
```
defineSymbol(math, main, rel, "\u2209", "\\notin", true);
```

I don't see anything wrong with that line of code. Maybe there is something wonky in metrics?

Reply:
Yep, 8713 is missing from `fontMetricsData.js`.  I've added an issue here: https://github.com/KaTeX/katex-fonts/issues/47